tests/test_utils.py

```diff
@@ -2,7 +2,8 @@
 from time import time
 from unittest import TestCase
 
-from sipa.utils import dict_diff, timetag_today
+from sipa.utils import dict_diff, timetag_today, meetingcal
+from base import AppInitialized
```

Comment:
I recommend using [relative imports](https://docs.python.org/3/reference/import.html#package-relative-imports) most of the time:
```python
from .base import AppInitialized
```
Or, if you have good reason to use an absolute import, it should be `tests.base`, because `tests` is the top-level package, and `base` a sub-package thereof.

…however I will delete this anyway because it is a leftover of the test you tried to create in e43d0accbfffa77c2495c655a7cdd4d4965ecfee  🙃

Reply:
…removing this will also fix the failed test BTW (see the `Checks` tab on this PR).
A great tool for finding unused imports among other things is `ruff`, I can try to set it up as a `pre-commit` hook if I have spare time.